Articles/WhatIs/WhatIsSoftwareProcessImprovement.md

```diff
@@ -0,0 +1,40 @@
+### What is Software Process Improvement?
+#### Contributed by  [Roscoe A. Bartlett](https://github.com/bartlett)
+#### Publication date: ???
+
+<!--- deck start --->
+Software process improvement is the application of a systematic approach to the identification, evaluation, and improvement of software processes which is critical for computational science and engineering projects.
+<!--- deck end --->
+
+<!--- body start --->
+
+The goal of **software process improvement** is a systematic methodology to improve the quality, efficiency, and effectiveness of software development and maintenance.
+Improvement efforts can implemented in a variety of ways, but it typically involve the following steps:
+
+1. **Identify the need for improvement:**  This can be done by conducting a process assessment, which involves gathering data on the current state of the software development process.
+
+2. **Establish goals and objectives:**  These goals should be specific, measurable, achievable, relevant, and time-bound.
+
+3. **Develop a plan:** This plan should include a timeline, resources, and responsibilities.
+
+4. **Implement the plan:** This involves implementing the changes (often couched as experiments) that have been identified and putting the collection of metrics in place to see if they are effective.
+
+5. **Monitor and evaluate the improvement effort:** This involves collecting metrics data on the effectiveness of the changes (i.e. the experiments) that have been implemented and making adjustments as needed based on feedback.
+
+By implementing software process improvement, organizations can:
```

Comment:
Implementing Software Process Improvement (SPI) can bring several benefits to organizations such as: